File: Data Structures & Algorithms/concatenation-of-array/submission-0.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        vector<int> res;
        int i=0;
        for(int i=0; i<2; i++)
            for(auto e: nums)
            {
                res.push_back(e);
            }
        return res;
    }
};

File: Data Structures & Algorithms/max-consecutive-ones/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number}
     */
    findMaxConsecutiveOnes(nums) {
        let result = 0;
        let count = 0;
        for (let num of nums) {
            count = num === 1 ? count + 1 : 0;
            result = Math.max(result, count);
        }
        return result;
    }
}
